### PR TITLE
Chemical sprites

### PR DESCRIPTION
**Pull request by binarysudoku**
## About the PR 

New sprites for chemgrenades, now with colourable reagent displays so you can see what you're about to be murdered by!

Plus some mini dmi refactors im sorry i got carried away D:

## Why's this needed? 

Legibility + eyecandy :)

## Changelog

```
(u)Erinexx:
(+)New chemgrenade sprites, now with visible coloured reagents!
```

### CODE REVIEW

**Comment by binarysudoku** [comment]
FUCK
DO NOT MERGE
I FUCKED UP THE CHEMICAL FILE
PLS HOLD

**Comment by binarysudoku** [comment]
Ok crisis averted

File: code/modules/chemistry/tools/grenades.dm [hunk 1]
```diff
@@ -23,6 +25,8 @@
 	stamina_crit_chance = 0
 
 	New()
+		fluid_image1 = image('icons/obj/chemical.dmi', "grenade-chem-fluid1", -1) //its 01:34 and im tired im sorry for this
+		fluid_image2 = image('icons/obj/chemical.dmi', "grenade-chem-fluid2", -1)
```
Comment (pali6) [inline code/modules/chemistry/tools/grenades.dm:29]:
```suggestion
		fluid_image1 = image('icons/obj/items/grenade.dmi', "grenade-chem-fluid1", -1)
		fluid_image2 = image('icons/obj/items/grenade.dmi', "grenade-chem-fluid2", -1)
```
Wrong icon.

Comment (binarysudoku) [reply code/modules/chemistry/tools/grenades.dm:29]: ahosfidjfdpg thank you pali <3

**Comment by binarysudoku** [comment]
> Nice, the chemical overlays are a neat addition. Could you make the neck part slightly shorter? It feels a bit too long when you look at an assembled grenade.

@flrsh could you clarify what you mean by the neck part on the assembled grenade? do you mean the fuse that hasnt been screwed in yet? If so i can fix that quickly

**Comment by pali6** [comment]
@flrsh Just checking if you haven't forgotten about the pending review.

**Pull request merged**